docs/querying/sql-functions.md

```diff
@@ -2473,28 +2575,92 @@ Shifts a timestamp forwards or backwards by a given number of time units.
 
 ## TIMESTAMP_TO_MILLIS
 
-`TIMESTAMP_TO_MILLIS(<TIMESTAMP>)`
+Returns the number of milliseconds since epoch for the given timestamp.
 
-**Function type:** [Scalar, date and time](sql-scalar.md#date-and-time-functions)
+* **Syntax:** `TIMESTAMP_TO_MILLIS(timestamp_expr)`
+* **Function type:** Scalar, date and time
+
+<details><summary>Example</summary>
+
+The following example converts the `__time` timestamp column from the `taxi-trips` datasource into milliseconds since epoch.
```

Comment:
```suggestion
The following example converts the `__time` column from the `taxi-trips` datasource into milliseconds since epoch.
```
We can omit the word "timestamp" to shorten the sentence.